cmd/terramate/e2etests/exp_vendor_test.go

```diff
@@ -132,6 +135,104 @@ func TestVendorModuleRecursive1DependencyIsPatched(t *testing.T) {
 	assert.EqualStrings(t, "", string(got))
 }
 
+func TestModVendorRecursiveMustPatchAlreadyVendoredModules(t *testing.T) {
+	// This reproduces the bug below:
+	//   - ModA depends on ModZ
+	//   - ModB depends on ModZ
+	// executing:
+	//   - tm vendor download ModA <ref>
+	//   - tm vendor download ModB <ref>
+	// will patch the ModA files to use ../modules/ModZ/...
+	// but will not for ModB because ModZ is already vendored.
```

Comment:
Can't we test this on the modvendor package instead of the e2etest ? the behavior/bug doesn't seem to be related to the CLI and could be tested directly on modvendor.Vendor no ? :thinking: (we get coverage + test doesn't depend on experimental CLI)

Reply:
the same for all those other comments.